src/control/README.md

```diff
@@ -54,6 +54,36 @@ For instructions on building and running DAOS see the [Quickstart guide](../../d
 
 * Run the tests `go test` within each directory containing tests
 
+### Run unit tests locally
+
+Checkout the DAOS source code:
+
+'git clone https://github.com/daos-stack/daos.git'
+
+Checkout the SPDK source code on branch v18.07.x:
+
+'git clone --single-branch --branch v18.07.x git@github.com:spdk/spdk.git'
+
+Continue installing SPDK with the procedure in the repository on branch v18.07.x: [SPDK-v18.07.x](https://github.com/spdk/spdk/tree/v18.07.x)
+
+Setup environment variables:
+
+'SPDK_REPO="/path/to/repo"'
+
+'export CGO_LDFLAGS="-L${SPDK_REPO}/build/lib"'
+
+'export CGO_CFLAGS=-I${SPDK_REPO}/include/'
+
+'export LD_LIBRARY_PATH="${SPDK_REPO}/build/lib:/daos/src/control/vendor/github.com/daos-stack/go-spdk/spdk"'
+
+Navigate into go-spdk directory in your DAOS repo:
+
+'cd ~/daos/src/control/vendor/github.com/daos-stack/go-spdk/spdk'
```

Comment:
I'd suggest making this a little less specific, instead of `~/daos` maybe something like `$DAOS_REPO` to indicate it is wherever the user cloned their DAOS repo.

Reply:
Sounds good. I'll include that fix.